integration/index.2.test.js

```diff
@@ -15,6 +18,58 @@ const {
   forceExitDaemon,
 } = require('@cnpmjs/rapid/lib/nydusd/nydusd_api');
 
+
+class Pids {
+  constructor(nodeModulesDir) {
+    this.nodeModulesDir = nodeModulesDir;
+  }
+
+  async getPsSnapshot() {
+    try {
+      const { stdout } = await exec('ps aux');
+      return stdout;
+    } catch (error) {
+      throw new Error(`Failed to execute 'ps aux': ${error.message}`);
+    }
+  }
+
+  async getPids() {
+    const pids = [];
+
+    try {
+      const snapshot = await this.getPsSnapshot();
+
+      const overlayPattern = new RegExp(`overlay.*?${this.nodeModulesDir}`, 'i');
+      const nfsPattern = new RegExp(
+        `/usr/local/bin/go-nfsv4.*?${this.nodeModulesDir}`, 'i'
+      );
+
+      snapshot.split('\n').forEach(line => {
+        if (overlayPattern.test(line)) {
+          const fields = line.split(/\s+/);
+          if (fields.length >= 11) {
```

Comment:
There seems to be a missing 'else' branch here. Also, should we specify the output format for 'ps' to prevent cross-platform compatibility issues?